### DISCUSSION TITLE
Issues or bugs

### DISCUSSION THREAD

**Discussion by coding-geek21** [coding-geek21/classDeck#51] [General]
If you are facing any issues or bugs related to project or while setting up the project in local pc ask here..

**Comment by xodervst** [comment]
I am facing error in Step 9: Installing Requirements. I am attaching the screenshot.
I have done all the previous steps properly. 
Virtual Environment is also activated.
[Click Here to get Image of The Error Message ](https://drive.google.com/file/d/17ZnmkBOjWFJZSAGchxx12VjO9XUe4WhZ/view?usp=sharing)

**Reply by coding-geek21** [reply]
- https://www.tutorialexample.com/fix-command-python-setup-py-egg_info-failed-with-error-code-1-python-tutorial/

- https://appuals.com/command-python-setup-py-egg_info/#:~:text=Fix%3A%20%E2%80%98Command%20%E2%80%9Cpython%20setup.py%20egg_info%E2%80%9D%20failed%20with%20error,...%204%20Try%20to%20Install%20the%20ez_setup.%20

Check out this maybe this will fix your error

**Reply by xodervst** [reply]
I tried both but it is saying requirements are already satisfied and the issue still exists.
